### PR TITLE
Fix structure

### PR DESCRIPTION
**Pull request by lauraluiz**
Please review @schleichardt ... just an overlook, there are too many files moved to see really the changes, probably.

Edit: Actually it's not that bad to check directly the files, ignoring the commits :)

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -22,12 +22,12 @@ notifications:
...
-deploy:
-  provider: heroku
-  buildpack: scala
-  api_key:
-    secure: NE6m0300BPmc8jJQbpTFyOP1uU6uK+7aVpN5lmKAPhV2gByzzq9XT3uwYqPcD9turCYujDXAmXrBzI6ISfLlrqpcxKjSupeUvvjpOxqhbpP65hA79VTuxDIajazzXrvjuKPgvUJAjD+1qdQWzoOjKJ4Y/iJ1YL+fqt8omOSFda0=
-  app: sphere-sunrise
-  on:
-    repo: sphereio/sphere-sunrise
-    branch: master
+#deploy:
```
Comment (schleichardt) [inline .travis.yml]: I would not outcomment stuff, it is dead code. You still can get it from history.

File: setup-widget/app/setupwidget/controllers/SetupController.java [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+    private Result onWidgetEnabled(final Supplier<Result> action) {
+        return isSettingsWidgetEnabled() ? action.get() : notFound();
+    }
+
+    private static void writeSettingsFile(String content) {
+        try {
+            Files.write(PATH, content.getBytes());
+            Logger.info("SPHERE.IO credentials saved in " + PATH.toString());
+        } catch (IOException e) {
+            throw new RuntimeException(e);
```
Comment (schleichardt) [inline setup-widget/app/setupwidget/controllers/SetupController.java]: Not necessary now, but I learned there is a special exception which you can use to wrap IOException: https://docs.oracle.com/javase/8/docs/api/java/io/UncheckedIOException.html

**Comment by schleichardt** [comment]
You can merge it @lauraluiz .

File: setup-widget/app/setupwidget/controllers/SetupController.java (hunk 2 above)
Comment (lauraluiz) [inline setup-widget/app/setupwidget/controllers/SetupController.java]: Nice to know!

**Pull request merged**